drivers/age-swift/Sources/age-swift/age.swift

```diff
@@ -0,0 +1,108 @@
+//
```

Comment:
Is there a standard AGE copyright header, including Apache license details?

Reply:
Will add this too